dashboards-observability/public/components/custom_panels/custom_panel_view.tsx

```diff
@@ -518,46 +520,48 @@ export const CustomPanelView = ({
       <EuiPage>
         <EuiPageBody component="div">
           <EuiPageHeader>
-            <EuiPageHeaderSection>
-              <EuiTitle size="l">
-                <h1>{openPanelName}</h1>
-              </EuiTitle>
-              <EuiFlexItem>
-                <EuiSpacer size="s" />
-              </EuiFlexItem>
-              Created on {moment(panelCreatedTime).format(UI_DATE_FORMAT)}
-            </EuiPageHeaderSection>
-            <EuiPageHeaderSection>
-              <EuiFlexGroup gutterSize="s">
-                {editMode ? (
-                  <>
+            {appMetrics || (
```

Comment:
Question: Is this for conditional rendering? If so should this be `&&`?

Reply:
This is for conditional rendering, yes. Is using `&&` recommended, `||` should just do the same thing just backwards as in the following will not render if `appMetrics` is true.